### PR TITLE
Update ug 1

### LINKED ISSUE #130: Load Command error message
Invalid message for loading a non-existing file is not relevant to what it displayed in the AB3

### LINKED ISSUE #133: Archive Command UG suggestion
Did not mention archive with the same file name will overwrite the existing file

### PR DESCRIPTION
**Pull request by Jianwen0451**
Current thing done

1. Change the Quickstart for non-tech users
2. Update the Warning of archive and load for a more suitable format
3. Change the introduction of the UserGuide 
4. A sidebar navigation is done
5. Add logo into UG and DG
6. close #133 by adding the warning of overwriting.
7. Update the success message from archive command
8. Update the error message when loading a non-existing file. Close #130

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,13 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+![Logo](images/StoreClass-Logo.png)
+
+<div markdown="block" class="alert alert-info">
+StoreClass (SC) is a desktop app for educators from private organizations e.g. tuition centers to manage their students. The educators will interact with the app through type while viewing the app through the window
```
Comment (tanyqcollin) [inline docs/UserGuide.md:9]: miss a full stop at the end.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -13,12 +19,16 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
+   - For Windows 11, see [here](https://www.youtube.com/watch?v=ykAhL1IoQUM)
+   - For MacOs, see [here](https://www.youtube.com/watch?v=lYKHFz8YaD4)
```
Comment (tanyqcollin) [inline docs/UserGuide.md:23]: The `MacOs` should be `MacOS`

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -261,10 +264,46 @@ _{more aspects and alternatives to be added}_
...
+
+![ArchiveToNExistingFile](images/ArchiveToExistingFile.png)
+
+The following sequence diagram illustrate how an archive operation is processed under `Logic` component.
+
+![ArchiveSequenceDiagram](images/ArchiveSequenceDiagram-Logic.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `ArchiveCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram. Some details like parameters of function is omitted for simplicity.
+
+Scenario 3 Loading from a file
```
Comment (yongqqqq) [inline docs/DeveloperGuide.md:295]: You can maybe format this by bolding the scenario or adding punctuations for a cleaner look.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -261,10 +264,46 @@ _{more aspects and alternatives to be added}_
...
+
+![ArchiveAndLoadInitialState](images/ArchiveAndLoadInitialState.png)
+
+Scenario 1 Archive to a new file
+
+In this scenario, the user is trying to archive the current address book into a new file named `archiveFile2.json`. He enters the command `archive pa/archiveFile2.json` A new file names `archiveFile2.json` will be created and hold the data of `addressBook.json`. And the data in `addressBook.json` will be discarded.
+
+![ArchiveToNewFile](images/ArchiveToNewFile.png)
+
+Scenario 2 Archive to a existing file
```
Comment (yongqqqq) [inline docs/DeveloperGuide.md:283]: You can maybe format this by bolding the scenario or adding punctuations for a cleaner look.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -261,10 +264,46 @@ _{more aspects and alternatives to be added}_
...
+#### The Following UML Object Diagrams will illustrate how archive and load are done
 
---------------------------------------------------------------------------------------------------------------------
+Before the archiving or loading
+
+We will use a simple case where there is one working AddressBook named `addressBook.json` in `data` folder and one archived AddressBook named `archivedFile1.json` in `archived` folder
+
+![ArchiveAndLoadInitialState](images/ArchiveAndLoadInitialState.png)
+
+Scenario 1 Archive to a new file
```
Comment (yongqqqq) [inline docs/DeveloperGuide.md:277]: You can maybe format this by bolding the scenario or adding punctuations for a cleaner look.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -174,9 +184,11 @@ The file name must ends with ".json" and must not contain any slash "/".
 
 There should be only one file name provided.
 
-#### Warning
+<div markdown="span" class="alert alert-primary">:rotating_light: **Warning:**
+All entries in the current address book will be discarded.
 
-All the entries in the current address book will be cleared.
+Archiving into a existing address book will overwrite the old address book.
```
Comment (yongqqqq) [inline docs/UserGuide.md:190]: It should be "an" existing address book.

File: docs/_layouts/default.html [hunk 7]
```diff
@@ -1,18 +1,88 @@
...
 
-    {%- include header.html -%}
+{%- include header.html -%}
 
-    <main class="page-content" aria-label="Content">
-      <div class="wrapper">
-        {{ content }}
-      </div>
-    </main>
+<!-- Flex container to hold sidebar and main content -->
```
Comment (yongqqqq) [inline docs/_layouts/default.html:10]: You could maybe make the sidebar retractable as it will block the main content on certain viewports.

**Review by yongqqqq — APPROVED** [review]
Overall, good job. Do consider the changes to be made for the user guide

File: docs/UserGuide.md (hunk 1 above)
Comment (Jianwen0451) [reply docs/UserGuide.md:9]: Fixed

File: docs/UserGuide.md (hunk 2 above)
Comment (Jianwen0451) [reply docs/UserGuide.md:23]: Fixed

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (Jianwen0451) [reply docs/DeveloperGuide.md:295]: Change to bold

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (Jianwen0451) [reply docs/DeveloperGuide.md:283]: Change to bold

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (Jianwen0451) [reply docs/DeveloperGuide.md:277]: Change to bold

File: docs/UserGuide.md (hunk 6 above)
Comment (Jianwen0451) [reply docs/UserGuide.md:190]: changed

File: docs/_layouts/default.html (hunk 7 above)
Comment (Jianwen0451) [reply docs/_layouts/default.html:10]: I am not very sure about html files. But I think this default is to place the side bar and main page side by side. So should not be blocking the main page

**Review by tanyqcollin — APPROVED** [review]

**Pull request merged**